examples/window/low_power.rs

```diff
@@ -79,18 +78,24 @@ fn update_winit(
             //     (e.g. the mouse hovers over a visible part of the out of focus window), a
             //     [`RequestRedraw`] event is received, or one minute has passed without the app
             //     updating.
-            WinitSettings {
-                focused_mode: bevy::winit::UpdateMode::reactive(Duration::from_secs(1)),
-                unfocused_mode: bevy::winit::UpdateMode::reactive_low_power(Duration::from_secs(5)),
-            }
+            WinitSettings::desktop_app()
         }
-        ApplicationWithRedraw => {
+        ApplicationWithRequestRedraw => {
             // Sending a `RequestRedraw` event is useful when you want the app to update the next
             // frame regardless of any user input. For example, your application might use
             // `WinitSettings::desktop_app()` to reduce power use, but UI animations need to play even
             // when there are no inputs, so you send redraw requests while the animation is playing.
             // Note that in this example the RequestRedraw winit event will make the app run in the same
             // way as continuous
+            let _ = redraw_request_events.send(RequestRedraw);
```

Comment:
Is the `let` binding needed here?
```suggestion
            redraw_request_events.send(RequestRedraw);
```